variables.tf

```diff
@@ -94,4 +94,14 @@ variable "ecs_patch_schedule" {
 variable "schedule_timezone" {
   default     = "America/New_York"
   description = "The timezone inside of which to run the patch windows. Defaults to US eastern."
-}
\ No newline at end of file
+}
+
+variable "stopped_ecs_task_log_group" {
+  default     = "ecs/stopped-tasks"
+  description = "The log group to use to log stopped task information."
+}
+
+variable "stopped_ecs_task_log_retention_days" {
+  default     = 30
```

Comment:
Should this default be lower? or even removed?

Reply:
we can start there and modify further